query-engine/driver-adapters/js/connector-test-kit-executor/src/index.ts

```diff
@@ -1,10 +1,22 @@
-import pgDriver from 'pg'
-import * as pg from '@prisma/adapter-pg'
 import * as qe from './qe'
 import * as engines from './engines/Library'
 import * as readline from 'node:readline'
 import * as jsonRpc from './jsonRpc'
-import {bindAdapter, ErrorCapturingDriverAdapter} from "@prisma/driver-adapter-utils";
+
+// pg dependencies
+import pgDriver from 'pg'
+import * as prismaPg from '@prisma/adapter-pg'
+
+// neon dependencies
+import { Pool as NeonPool, neonConfig } from '@neondatabase/serverless'
+import { WebSocket } from 'undici'
+import * as prismaNeon from '@prisma/adapter-neon'
+neonConfig.webSocketConstructor = WebSocket
+
+import {bindAdapter, DriverAdapter, ErrorCapturingDriverAdapter} from "@prisma/driver-adapter-utils";
+
+const SUPPORTED_ADAPTERS: Record<string, (_ : string) => Promise<DriverAdapter>>
+    = {'pg': pgAdapter, 'neon:': neonAdapter} as const;
```

Comment:
Extra colon inside `'neon:':`.

`as const` can be removed because the type of the key is immediately widened back to `string` via the explicit type annotation.

```suggestion
    = {pg: pgAdapter, neon: neonAdapter};
```